wct.conf.json

```diff
@@ -3,7 +3,7 @@
     "local": {
       "browsers": ["chrome"],
       "browserOptions": {
-        "chrome": ["headless", "disable-gpu", "no-sandbox"]
+        "chrome": ["disable-gpu", "no-sandbox"]
```

Comment:
Curious about whether there's a performance penalty we pay for not running headless. I guess if it's just for the local tests it's not a big deal. If it ends up getting costly we could always split our tests into 2 sets -- one that runs headlessly and one that doesn't.

Reply:
It's probably a bit slower but it runs in a few seconds right now. Agreed about splitting should we need to.